### PR TITLE
Enabled Instna Support with traceloop

### PR DESCRIPTION
**Pull request by gyliu513**
Fixed https://github.com/traceloop/openllmetry/issues/342

@nirga ^^

### CODE REVIEW

**Comment by nirga** [comment]
Thanks @gyliu513! 2 things:
* You need to update the `mint.json` file with the new doc you added. 
* Run `npx mintlify@latest dev` to see that everything looks OK.

File: openllmetry/integrations/instana.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Add following to the file:
+
+```
+com.instana.plugin.opentelemetry:
+  enabled: true
+  grpc:
+    enabled: true
+  http:
+    enabled: true
```
Comment (nirga) [inline openllmetry/integrations/instana.mdx:27]: You don't need HTTP since you're configuring traceloop to send OTLP/GRPC

Comment (gyliu513) [reply openllmetry/integrations/instana.mdx:27]: My bad, I should remove that, thanks!

**Comment by gyliu513** [comment]
doc looks great, thanks @nirga 

Screenshot 2024-01-24 at 11 44 42 AM

**Review by nirga — APPROVED** [review]

**Pull request merged**